relayer/strategies.go

```diff
@@ -83,8 +83,8 @@ func relayerListenLoop(src, dst *Chain, doneChan chan struct{}, sh *SyncHeaders,
 
 	// Start client for source chain
 	if err = src.Start(); err != nil {
-		src.Error(err)
-		return
+		// Don't exit on error, if the client is already started its ok
+		// TODO: Need to do anything here?
```

Comment:
Could we perhaps perform a check on the error to confirm if it is really started or if something else occured?